code/components/citizen-server-gui/src/ServerGui.cpp

```diff
@@ -193,9 +193,12 @@ devgui_convar "Tools/Network/State/Player List" svplayerlist
 						}
 					}
 
+					// suffix 'txAdmin' if we're the txAdmin child process
+					auto txSuffix = console::GetDefaultContext()->GetVariableManager()->FindEntryRaw("txAdminServerMode") ? "/txAdmin" : "";
```

Comment:
This is something that might be best factored out into a helper somewhere given we're doing this in a few places.